catalog/core/catalog-core-directorymonitor/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -21,10 +21,12 @@
            http://aries.apache.org/blueprint/xmlns/blueprint-cm/v1.1.0
            http://aries.apache.org/schemas/blueprint-cm/blueprint-cm-1.1.0.xsd">
 
-    <reference id="camelContext" interface="org.apache.camel.CamelContext"
+    <!-- Over blueprint timeout is changed from 5 minutes to 30 minutes (see pom.xml) -->
+    <!-- Set these 2 references back to the default 5 minutes  -->
+    <reference timeout="300000" id="camelContext" interface="org.apache.camel.CamelContext"
```

Comment:
❓ Why did the reference timeouts for the AttributeRegistry and CamelContext need to be raised?